bridge/package.json

```diff
@@ -15,8 +15,12 @@
     "lint:fix": "eslint --fix ./",
     "prettier:check": "prettier --check ./**/*.{js,ts,json,html}",
     "prettier:fix": "prettier --write ./**/*.{js,ts,json,html}",
-    "cypress:open": "start-server-and-test 'yarn start:ci' http://localhost:3000 'cypress open'",
-    "cypress:run": "cypress run --browser=chrome",
+    "cypress:open": "run-script-os",
+    "cypress:open:win32": "set TZ=Europe/Berlin && start-server-and-test 'yarn start:ci' http://localhost:3000 'cypress open'",
+    "cypress:open:default": "TZ=Europe/Berlin start-server-and-test 'yarn start:ci' http://localhost:3000 'cypress open'",
+    "cypress:run": "run-script-os",
+    "cypress:run:win32": "set TZ=Europe/Berlin && yarn cypress run --browser=chrome",
+    "cypress:run:default": "TZ=Europe/Berlin cypress run --browser=chrome",
```

Comment:
I would suggest to just handle it like this and just add developer documentation in the README:
```
    "cypress:open:win": "set TZ=Europe/Berlin && start-server-and-test 'yarn start:ci' http://localhost:3000 'cypress open'",
    "cypress:open:unix": "TZ=Europe/Berlin start-server-and-test 'yarn start:ci' http://localhost:3000 'cypress open'",
    "cypress:run:win": "set TZ=Europe/Berlin && yarn cypress run --browser=chrome",
    "cypress:run:unix": "TZ=Europe/Berlin cypress run --browser=chrome",
    "test:ui": "start-server-and-test 'yarn start:ci' http://localhost:3000 'yarn cypress:run:unix'"
```